### PR TITLE
add par upload car ca marche pas autrement

### PR DESCRIPTION
**Pull request by loumir**

### CODE REVIEW

**Comment by loumir** [comment]
voici les motifs pour la partie 1 et 2 . je n ai pas encore regardé le modèle . 
Mireille

**Comment by olebole** [comment]
Could discussion and git messages happen in english? Other languages are hard to follow for the general public.

**Comment by lmichel** [comment]
Sorry, this brief discussion shouldn't have landed on that thread.

**Comment by olebole** [comment]
#7 is the same... It would be good to have a message (and/or title) in English there and here what the purpose of these Pull Requests was. Even Google Translate doesn't really help here.
